docs/machineset-gcp.md

```diff
@@ -0,0 +1,72 @@
+# Creating a GCP Windows MachineSet
+
+_\<windows_server_image \>_ should be replaced with the full path to an image of a [supported version](wmco-prerequisites.md#supported-windows-server-versions) of Windows.
+If using a standard image, this will be of the form: `projects/windows-cloud/global/images/windows-server-<VERSION>`.
+
+_\<infrastructure_id\>_ should be replaced with the output of:
+```shell script
+ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+```
+_\<region\>_ should be replaced with a region, such as `us-central1`.
+
+_\<zone\>_ should be replaced with a zone within the chosen region, such as `us-central1-a`.
+
+_\<zone_suffix\>_ should be replaced with the suffix of the chosen zone, such as `a`.
+
+_\<project_id\>_ should be replaced with the GCP project this cluster was created under.
+```
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
+  name: <infrastructure_id>-windows-worker-<zone_suffix>
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-windows-worker-<zone_suffix>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-windows-worker-<zone_suffix>
+        machine.openshift.io/os-id: Windows
+    spec:
+      metadata:
+        labels:
+          node-role.kubernetes.io/worker: ""
+      providerSpec:
+        value:
+          apiVersion: machine.openshift.io/v1beta1
+          canIPForward: false
+          credentialsSecret:
+            name: gcp-cloud-credentials
+          deletionProtection: false
+          disks:
+          - autoDelete: true
+            boot: true
+            image: 
```

Comment:
Image is missing from the template, in my case I had something like:
`
image: projects/windows-cloud/global/images/<gcp_windows_image>`

Although I guess that could be different depending on the account, however I think it might be useful to define it.